### PR TITLE
#263: reduce execution time for test `test_shrink_cache_if_more_then_10_mb`

### LINKED ISSUE #263: `test_shrink_cache_if_more_then_10_mb` is too slow
This test is very slow (over 4s on my Mac). Let's make it as fast as all others (below 100ms).

### PR DESCRIPTION
**Pull request by Yegorov**
Before
```
SqliteStoreTest
  test_shrink_cache_if_more_then_10_mb                            PASS (1.51s)
```

After
```
SqliteStoreTest
  test_shrink_cache_if_more_then_10_mb                            PASS (0.05s)
```

Closes #263 

## Summary by CodeRabbit

* **Tests**
  * Updated boundary-condition scenarios for the SQLite-backed store to use larger values and very long keys across sequential writes.
  * Reduced iteration counts while keeping data volume sufficient to hit size thresholds for write skipping and cache shrinking.
  * Test data now uses 8 KB values and 52 successive keys, including an initial extremely long key, to reflect boundary conditions.
  * No changes to user-facing behavior.

### CODE REVIEW

File: test/fbe/middleware/test_sqlite_store.rb [hunk 1]
```diff
@@ -155,16 +155,16 @@ def test_skip_write_if_value_more_then_10k_bytes
...
           store.write('b', 'bb')
           store.write('c', 'cc')
         end
         assert_equal('cc', store.read('c'))
         Time.stub(:now, rand((Time.now - (5 * 60 * 60))..Time.now).round) do
-          value = SecureRandom.alphanumeric(2048)
-          10_240.times do
+          key = 'a' * 65_536
+          value = SecureRandom.alphanumeric(8_192)
+          52.times do
```
[the remark this answers is not in the document]
Comment (Yegorov) [reply test/fbe/middleware/test_sqlite_store.rb:167]: @coderabbitai yes, but in structure db we have other fields: created_at, touched_at. Also we have indexes. The final file size is more than 10 MB. This can be verified by this [assert](https://github.com/zerocracy/fbe/blob/61249e0dddaac2e95acdf1ea51746a63347f6aef/test/fbe/middleware/test_sqlite_store.rb#L173). If i increase iteration count to 150 I will also increase the test time.

**Comment by yegor256** [comment]
@Yegorov cool, thanks!

**Comment by 0crat** [comment]
@yegorov Hey there! 👋 Nice work on your contribution! You've earned +4 points for this one. Here's the breakdown: +16 for showing up, -16 because we missed the code review (let's fix that next time!), and -8 because we're aiming for at least 20 hits-of-code (you had 6). We bumped you up +12 to keep you motivated. Remember, more code and reviews can boost your points even higher! Your total is now [+71](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep the [code flowing](https://www.yegor256.com/2018/03/06/speed-vs-quality.html), and don't forget to check your [Zerocracy balance](https://www.zerocracy.com/)!
